docs/howto/create-a-datasource/configuring-the-jdbc-driver-manually.adoc

```diff
@@ -28,6 +28,7 @@
 +
 In the `thorntail` -> `datasources` -> `jdbc-drivers` tree, define the driver by
 key, which will be used later for connecting it to the datasource.
+Set the `driver-module-name` to that defined by the `module.xml` from earlier.
```

Comment:
Please replace _earlier_ with an explicit reference to a file/config/point in time where the variable was defined. We try to avoid ambiguity as much as possible.

Reply:
How do I refer to the prereq step from this doc?